src/seedu/addressbook/data/tag/Tagging.java

```diff
@@ -0,0 +1,31 @@
+package seedu.addressbook.data.tag;
+import seedu.addressbook.data.person.Person;
+
+public class Tagging {
+	
+	public static final String STATE_ADD_TAG = "+";
+	public static final String STATE_REMOVE_TAG = "-";
+	
+	private Tag tag;
+	private Person contact;
+	private String state;
```

Comment:
Using string is a risky choice to keep track of the addition/deletion as it is very error-prone .
 You can consider the alternative of defining a enumeration for OperationType (e.g.) with "ADD" and "DELETE" as values.